content/articles/2018-04-readxl-1-1-0.Rmd

```diff
@@ -0,0 +1,154 @@
+```
+
+We are pleased to announce that [readxl](http://readxl.tidyverse.org) 1.1.0 is now available on CRAN. Learn more about readxl at <http://readxl.tidyverse.org>.
+
+The readxl package makes it easy to get tabular data out of Excel files and into R with code, not mouse clicks. It supports both the legacy `.xls` format and the modern XML-based `.xlsx` format. readxl is expressly designed to be easy to install and use on all operating systems. Therefore it has no external dependencies, such as Java or Perl, which have historically been a source of aggravation with some R packages that read Excel files. 
+
+The easiest way to install the latest version from CRAN is to install the whole tidyverse.
+
+```{r eval = FALSE}
+install.packages("tidyverse")
+```
+
+Alternatively, install just readxl from CRAN:
+
+```{r eval = FALSE}
+install.packages("readxl")
+```
+
+Regardless, you will still need to load readxl explicitly via `library(readxl)`, because it is not a core tidyverse package loaded via `library(tidyverse)`. Do this like so in your scripts:
+
+```{r eval = FALSE}
+library(readxl)
+```
+
+readxl was last released almost exactly one year ago, in April 2017, at version 1.0.0. The associated [blog post](http://readxl.tidyverse.org/articles/releases/readxl-1.0.0.html) summarizes many nifty new features in version 1.0.0. In contrast, version 1.1.0 is considerably less exciting for most users, but includes two important improvements:
+
+  * Security patches in the embedded libxls library
+  * Better support for detecting the format of files, i.e. whether to read as xls or xlsx
+  
+Version 1.1.0 also includes several other small features and bug fixes. For full details on all versions, see the [changelog](http://readxl.tidyverse.org/news/index.html).
+
+Read (or skip!) to the end for a story about how readxl was used in a recent Los Angeles Times piece about homeless arrests. Or, rather, was not used.
+
+## Security fixes for libxls
+
+readxl embeds the library [libxls](https://github.com/evanmiller/libxls) in order to read xls files. In fall 2017, three security vulnerabilities were identified in libxls and were shown to affect readxl:
+
+  * [CVE-2017-12110](https://www.talosintelligence.com/vulnerability_reports/TALOS-2017-0462)
+  * [CVE-2017-2896](https://www.talosintelligence.com/vulnerability_reports/TALOS-2017-0403)
+  * [CVE-2017-2897](https://www.talosintelligence.com/vulnerability_reports/TALOS-2017-0404)
+  
+Those have now been addressed upstream and version 1.1.0 of readxl embeds a version of libxls that includes those fixes.
+
+FYI, there are two other CVEs for libxls that have long been resolved in libxls itself, that do not affect readxl, have never affected readxl, but nonetheless still mention readxl in their text. [CVE-2017-12111](https://www.talosintelligence.com/vulnerability_reports/TALOS-2017-0463), [CVE-2017-2919](https://www.talosintelligence.com/vulnerability_reports/TALOS-2017-0426)
```

Comment:
Maybe reword to something like

Two other CVEs mention readxl, but actually concern parts of libxls unused by readxl ([CVE-2017-12111], [CVE-2017-2919])